### PR TITLE
adding rubocop support to example plugin

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by ohadlevy** [comment]
fixes #7 /cc @domcleal

**Comment by domcleal** [comment]
Yes, yes, I saw... patience.

File: lib/tasks/foreman_plugin_template_tasks.rake [hunk 1]
```diff
@@ -10,22 +10,39 @@ end
...
+end
+
 Rake::Task[:test].enhance do
   Rake::Task['test:foreman_plugin_template'].invoke
 end
 
 load 'tasks/jenkins.rake'
 if Rake::Task.task_defined?(:'jenkins:unit')
-  Rake::Task["jenkins:unit"].enhance do
+  Rake::Task['jenkins:unit'].enhance do
```
Comment (domcleal) [inline lib/tasks/foreman_plugin_template_tasks.rake]: also needs to call the newly defined rubocop task

**Comment by ohadlevy** [comment]
@domcleal updated.

**Comment by domcleal** [comment]
I'm getting a couple of warnings, do you?

<pre>
✔ ~/code/foreman/foreman_test [:163cf4c|✚ 23…18] 
15:12 $ rubocop 
/home/dcleal/code/foreman/foreman_test/.rubocop.yml: Style/MethodLength has the wrong namespace - should be Metrics
Inspecting 16 files
........C.......

Offenses:

config/routes.rb:2:1: C: Extra empty line detected at block body beginning.
config/routes.rb:4:1: C: Extra empty line detected at block body end.

16 files inspected, 2 offenses detected
✘-1 ~/code/foreman/foreman_test [:163cf4c|✚ 23…18] 
15:12 $ rubocop --version
0.28.0
</pre>

**Comment by ohadlevy** [comment]
@domcleal updated, it seems it depended on my ruby version /gemsets, thanks, do you know how to fix the warning:
```.rubocop.yml: Style/MethodLength has the wrong namespace - should be Metrics```

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Don't enforce documentation
+Style/Documentation:
+  Enabled: false
+
+# Force before_filter until upgrade to Rails 4
+Rails/ActionFilter:
+  EnforcedStyle: filter
+
+Style/MethodLength:
```
Comment (domcleal) [inline .rubocop.yml]: s/Style/Metrics/

**Comment by ohadlevy** [comment]
@domcleal pushed. thanks :)

**Comment by domcleal** [comment]
Thanks @ohadlevy!

**Pull request merged**